src/couch_mrview/test/eunit/couch_mrview_ddoc_updated_tests.erl

```diff
@@ -88,7 +88,7 @@ check_indexing_stops_on_ddoc_change(Db) ->
     ?_test(begin
         DDocID = <<"_design/bar">>,
 
-        IndexesBefore = get_indexes_by_ddoc(DDocID, 1),
+        IndexesBefore = get_indexes_by_ddoc(Db#db.name, DDocID, 1),
```

Comment:
Notice some of the tests were failing with:

```
[2021-12-03T12:45:35.668Z] /home/****/workspace/****-cm1_PullRequests_PR-3860/21.3.8.22/build/apache-couchdb-3.2.1-9904993/src/couch_mrview/test/eunit/couch_mrview_ddoc_updated_tests.erl:91: record db undefined

[2021-12-03T12:45:35.668Z] /home/****/workspace/****-cm1_PullRequests_PR-3860/21.3.8.22/build/apache-couchdb-3.2.1-9904993/src/couch_mrview/test/eunit/couch_mrview_ddoc_updated_tests.erl:130: record db undefined
```

We probably want to use an accessor function there `couch_db:name(Db)`

Reply:
hm, all the tests passed for me...

Reply:
`couch_mrview` ones fail for me:

```
make eunit apps=couch_mrview
==> couch_mrview (compile)
==> rel (compile)
==> asf-3 (compile)
WARN:  Missing plugins: [covertool]
==> asf-3 (setup_eunit)
Writing tmp/etc/default_eunit.ini
Writing tmp/etc/local_eunit.ini
Writing tmp/etc/eunit.ini
Writing tmp/etc/vm.args
==> couch_mrview (eunit)
Compiled src/couch_mrview_show.erl
Compiled src/couch_mrview_test_util.erl
Compiled src/couch_mrview.erl
Compiled src/couch_mrview_util.erl
Compiled src/couch_mrview_updater.erl
Compiled src/couch_mrview_cleanup.erl
Compiled src/couch_mrview_update_notifier.erl
Compiled src/couch_mrview_index.erl
/home/nickv/asf-3/src/couch_mrview/test/eunit/couch_mrview_ddoc_updated_tests.erl:91: record db undefined
/home/nickv/asf-3/src/couch_mrview/test/eunit/couch_mrview_ddoc_updated_tests.erl:130: record db undefined
Compiling /home/nickv/asf-3/src/couch_mrview/test/eunit/couch_mrview_ddoc_updated_tests.erl failed:
ERROR: eunit failed while processing /home/nickv/asf-3/src/couch_mrview: rebar_abort
make: *** [Makefile:174: eunit] Error 1
```